portfolio/src/main/webapp/index.html

```diff
@@ -67,36 +67,29 @@ <h3>Photography</h3>
         <div class="photo-container">
           <img src="images/luke_skywalker_figure.jpg" alt="Luke Skywalker using the force">
           <p>
-            Looking around my desk, I was inspired by the figurines that give 
-            my workspace character. 
-            When I looked at the fortune cookie I noticed how beautful 
-            reflected light can be and the magical experience it gives. 
-            When I prepared to take a picture of the fortune cookie I noticed 
-            that the reflect light in front of Luke Skywalker makes it seem 
-            like he is using the force.
+            Looking around my desk, I was inspired by the figurines that give my workspace character. 
+            When I looked at the fortune cookie I noticed how beautful reflected light can be and the magical experience it gives. 
+            When I prepared to take a picture of the fortune cookie I noticed that the reflect light in front of Luke Skywalker makes it seem like he is using the force.
```

Comment:
Try to keep your lines to 100 characters or less to make it easier to view the entire line

Reply:
Yup sorry about that, the line is shortened in the Google Cloud Shell editor automatically, but I have to indent it in order for it to show on the actual file.